### PR TITLE
[#197][#198] Add `new_project.kts` and support arguments

### PR DESCRIPTION
**Pull request by Tuubz**
#197 #198 

## What happened 👀

- Add `new_project.kts` inside `/scripts`
- Add support for arguments and store in variables for later use

## Proof Of Work 📹

Arguments can be printed in the logs:

https://user-images.githubusercontent.com/18277915/167112085-b2427bef-f10b-47d0-898b-388524bca232.mov

### CODE REVIEW

File: newproject.kts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+object NewProject {
```
Comment (manh-t) [inline newproject.kts:1]: should we move `newproject.kts` to a directory? Maybe we will have `test.kts` for verifying `newproject.kts` script later 😄

**Review by manh-t — COMMENTED** [review]
rest LGTM

File: newproject.kts (hunk 1 above)
Comment (Tuubz) [reply newproject.kts:1]: I suppose we could move it into `/scripts` or `/script`, what do you think? 🙏

Comment (manh-t) [reply newproject.kts:1]: @Tuubz any of them is okay 🚀

File: newproject.kts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    fun generate(args: Array<String>) {
+        handleArguments(args)
+    }
+
+    private fun handleArguments(args: Array<String>) {
+        args.forEach {
+            val (key, value) = it.split(ARGUMENT_DELIMITER)
+            if (key == KEY_APP_NAME) appName = value
+            if (key == KEY_PACKAGE_NAME) packageName = value
```
Comment (lydiasama) [inline newproject.kts:18]: Can we use when(key) for readability?

**Comment by Tuubz** [comment]
Updated 🚀

**Review by chornerman — APPROVED** [review]

**Review by Wadeewee — APPROVED** [review]

**Review by manh-t — APPROVED** [review]

**Comment by Tuubz** [comment]
@kaungkhantsoe Even tho we already have three approvals, I'll wait for your approval before merging 🚀

**Review by lydiasama — APPROVED** [review]

**Review by minhnimble — APPROVED** [review]

**Review by kaungkhantsoe — APPROVED** [review]
LGTM 🚀

**Pull request merged**